app/cmd/preview.go

```diff
@@ -100,10 +128,13 @@ var previewCmd = &cobra.Command{
 		fmt.Println("\nPlease wait while Learn builds your preview...")
 
 		// Start a processing spinner that runs until Learn is finsihed building the preview
-		s := spinner.New(spinner.CharSets[32], 100*time.Millisecond)
-		s.Color("green")
+		s = spinner.New(spinner.CharSets[32], 100*time.Millisecond)
+		s.Color("blue")
```

Comment:
ooh new colorssss

Reply:
yeah well now we have these little green checks for each step so changed the colors on that loader